remotes/docker/fetcher.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/containerd/containerd/images"
 	"github.com/containerd/containerd/log"
 	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
```

Comment:
I'd rather not introduce `k8s.io/apimachinery` as dependency for this package, as this code is also used outside of projects using kubernetes

Reply:
ok, I will try to do as before.